packages/components/icon/src/assets/icons/menu-switch.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Could you please optimize this svg in some tool like svgo:
https://jakearchibald.github.io/svgomg/

It will save around 33% of the size without quality loss:
<img width="1167" alt="Screenshot 2020-06-02 at 12 19 06" src="https://user-images.githubusercontent.com/55530374/83509186-59caca00-a4cb-11ea-9133-613602f8bc60.png">

Reply:
Done 👍